app/assets/javascripts/oxalis/view/layouting/layout_persistence.js

```diff
@@ -24,32 +24,32 @@ const localStorageKeys = {
 function readStoredLayoutConfigs() {
   const storedLayoutVersion = localStorage.getItem(localStorageKeys.currentLayoutVersion);
   if (!storedLayoutVersion || disableLayoutPersistance) {
-    return defaultLayoutSchema;
+    return getDefaultLayoutSchema();
   }
   const layoutString = localStorage.getItem(localStorageKeys.goldenWkLayouts);
   if (!layoutString) {
-    return defaultLayoutSchema;
+    return getDefaultLayoutSchema();
   }
   try {
     const version = JSON.parse(storedLayoutVersion);
     const layouts = JSON.parse(layoutString);
     if (currentLayoutVersion > version) {
       if (version !== 5) {
-        return defaultLayoutSchema;
+        return getDefaultLayoutSchema();
       }
       // migrate to newset schema
       const withMulipleLayoutsSchema = {
         OrthoLayoutView: {
-          "Custom Layout": layouts.OrthoLayoutView || defaultLayouts.OrthoLayoutView,
+          "Custom Layout": layouts.OrthoLayoutView || getDefaultLayouts().OrthoLayoutView,
```

Comment:
Please store `getDefaultLayouts()` in a variable (above line 40) and use that within the four occurrences here (I know, that the method call is memoized, but this is not clear to the reader of this function and also this might change in the future).